mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -197,7 +197,8 @@ CLASS age net/minecraft/entity/Entity
 	METHOD bE getComponentHoverEvent ()Ljc;
 	METHOD bF getBoundingBox ()Lckc;
 	METHOD bH getEyeHeight ()F
-	METHOD bL getCommandStats ()Lbam;
+	METHOD bJ getPos ()Les;
```

Comment:
getBlockPos, or maybe getPosVec on the one below. Do look into how bK interacts with this.